docs/docs/reference/built-in-components/gatsby-script.md

```diff
@@ -4,16 +4,18 @@ title: Gatsby Script API
 
 > Support for the Gatsby Script API was added in `gatsby@4.15.0`.
 
-Gatsby includes a built-in `<Script>` component that aids in loading third-party scripts performantly.
+Gatsby includes a built-in `<Script>` component that aids in loading scripts performantly.
 
-It offers a convenient way to declare different loading strategies and a default loading strategy that gives Gatsby users strong performance out of the box.
+It offers a convenient way to declare different [loading strategies](#strategies), and a default loading strategy that gives Gatsby users strong performance out of the box. It supports both [scripts with sources](#scripts-with-sources) and [inline scripts](#inline-scripts).
+
+The Gatsby `<Script>` component is a great tool to reach for when you want **maximum flexibility** and **fine-grained control** over how scripts load on your Gatsby site.
```

Comment:
I think this could scare people away from using it? Perhaps mention its useful in situations both situations:
- you don't want to manage scripts at all besides importing them
- you want total control and need the flexibility

Reply:
Thanks, updated!